.github/workflows/uffizzi-build.yaml

```diff
@@ -0,0 +1,87 @@
+name: Build PR Image
+on:
+  pull_request:
+    types: [opened,synchronize,reopened,closed]
+
+jobs:
+
+  build-cockpit:
+    name: Build and Push `cockpit`
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name != 'pull_request' || github.event.action != 'closed' }}
+    outputs:
+      tags: ${{ steps.meta.outputs.tags }}
+    steps:
+      - name: Checkout git repo
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
I had a look at https://github.com/docker/setup-buildx-action , but this seems very generic. It wraps https://github.com/docker/buildx, which again wraps https://github.com/moby/buildkit, i.e. this very quickly gets to the point when this is completely non-understandable nor reproducible locally, and presumably will also take a lot longer than necessary. We already know how to build cockpit, and should re-use that here.

I'm afraid I don't yet know anything about the uffizi environment (not even whether it's deb, rpm, arch, or something else), so that will take some time to learn.

Did you test this on a fork? How do these environments look like?

Thanks!

Reply:
How do you log in there? I.e. which user/password/host? I don't see this on https://github.com/UffizziCloud/uffizzi. So far it looks like it's running a cockpit/ws container in bastion mode, but that's usually not the interesting part. Thanks!

Reply:
@waveywaves : I'm afraid I don't understand your answer. Right now, on https://app.uffizzi.com//github.com/waveywaves/cockpit/pull/1 I see the cockpit login page. In order to to see the actual cockpit session, one  needs to log into the machine which has the packages/pages from the PR installed. Otherwise, the only thing you could test was the login page itself (which doesn't change often, and isn't very interesting). So what user/password do I need to type there to log into the temporary server?